### ISSUE TITLE
Bug: error summarizing team prompt

### ISSUE THREAD

**Issue by Dschoordsch** [ParabolInc/parabol#10299]
https://parabol.sentry.io/issues/5929114736/?referrer=github_plugin

```
error: syntax error at or near "where"
  File "app:///gqlExecutor.js", line 1185732, in Parser.parseErrorMessage
  File "app:///gqlExecutor.js", line 1185571, in Parser.handlePacket
  File "app:///gqlExecutor.js", line 1185484, in Parser.parse
  File "app:///gqlExecutor.js", line 1185267, in TLSSocket.<anonymous>
  File "app:///gqlExecutor.js", line 1367686, in PostgresConnection.executeQuery
...
(12 additional frame(s) were not displayed)
```

**Comment by Dschoordsch** [comment]
This might cause team prompt summaries to not being sent out.

**Comment by Dschoordsch** [comment]
Already fixed on master by #10295
